CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ set(OPENEXR_VERSION_API "${OpenEXR_VERSION_MAJOR}_${OpenEXR_VERSION_MINOR}")
 #   2. API added:     CURRENT+1.0.AGE+1
 #   3. API changed:   CURRENT+1.0.0
 #
-set(OPENEXR_LIBTOOL_CURRENT 29)
+set(OPENEXR_LIBTOOL_CURRENT 31)
```

Comment:
Is this after our ... discussion? ... questioning? ... hemming and hawing about libtool? Or did we already change something where this change is warranted?